library/src/main/java/io/constructor/core/ConstructorIo.kt

```diff
@@ -105,6 +106,26 @@ object ConstructorIo {
         return dataManager.getAutocompleteResults(query, params.toTypedArray())
     }
 
+    fun search(text: String, vararg facets: Pair<String, List<String>>, page: Int? = null, perPage: Int? = null, groupId: Int? = null): Observable<ConstructorData<SearchResponse>> {
+        val sessionId = preferenceHelper.getSessionId(sessionIncrementEventHandler)
+        val encodedParams: ArrayList<Pair<String, String>> = arrayListOf()
+        val params: ArrayList<Pair<String, String>> = arrayListOf()
```

Comment:
We create and add items to `params`, but I cannot see where it is read from.

Reply:
good point, its a bug we should put it to encoded params, as we build serach url dynamically due to shortcomings of http lib we use